### PR TITLE
07 15 updates

### PR DESCRIPTION
**Pull request by TresTres**
YEEEEEEEEEEEEEEEEEEEEEEER

### CODE REVIEW

File: weekly-updates/project-lead-meetings/07_15_18.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+_Divide your project into three or four parts, each being an important component_:
+
+_What are the prerequisites for an architect working on your codebase? (Keep the components above in mind)_:
+
+_Describe your project in one sentence - 20 or so words max_:
+
+## front end
+
+_UPDATES_: Login management is now done with separated UserForm and LoginForm components.  Main app connected to Redux store and tested to some extent.  Currently working on the cesspool known as the original `UserForm.js`
```
Comment (hemangandhi) [inline weekly-updates/project-lead-meetings/07_15_18.md]: "Cesspool" :hankey: my kinda spice

File: weekly-updates/project-lead-meetings/07_15_18.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+_QUESTIONS_: Working on Cookies through middleware, any suggestions would be considered.  If any documentation already exists for front-end, please let me know, otherwise I'm writing some for the sake of tracking what I'm doing.  I'll have to make new docs after it's all finished.
```
Comment (hemangandhi) [inline weekly-updates/project-lead-meetings/07_15_18.md]: Is this on `bastard`?

File: weekly-updates/project-lead-meetings/07_15_18.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+_Divide your project into three or four parts, each being an important component_:
+
+Refactor React components to Redux (splitting away actions and logic to proper files and moving from component-specific states to a unified state)
+
+Fix all open issues with App, Events, and Slack
+
+Introduce robust testing and continuously test until the day-of
+
+Introduce other features that suddently punch us in the face
+
```
Comment (hemangandhi) [inline weekly-updates/project-lead-meetings/07_15_18.md]: Beautiful.

File: weekly-updates/project-lead-meetings/07_15_18.md (hunk 2 above)
Comment (TresTres) [reply weekly-updates/project-lead-meetings/07_15_18.md]: Yes

**Pull request merged**